src/serialization/index.ts

```
@@ -0,0 +1,691 @@
+		}
+		
+		xml += '>'
+		
+		if (options.includeTextContent !== false && element.textContent) {
+			xml += escapeXML(element.textContent)
+		}
+		
+		xml += `</${element.tagName.toLowerCase()}>`
+		
+		return xml
+	})
```

Biome (2.1.2)

[error] This generator function doesn't contain yield.



(lint/correctness/useYield)